### ISSUE TITLE
opensuse/system-libdrm.patch suitable for portable build ?

### ISSUE THREAD

**Issue by intika** [Eloston/ungoogled-chromium#571]
@Eloston 

In my custom build i add this patch to the portable build (patch_order.list)
opensuse/system-libdrm.patch

Is it suitable to add that to portable release mainstream ?

**Comment by Eloston** [comment]
Why would you need that patch in your builds of Portable Linux?

**Comment by intika** [comment]
**What the patch does :**
```
--- a/ui/gfx/native_pixmap_handle.cc
+++ b/ui/gfx/native_pixmap_handle.cc
@@ -12,6 +12,9 @@
 namespace gfx {
 
 #if defined(OS_LINUX)
+#ifndef DRM_FORMAT_MOD_INVALID
+#define DRM_FORMAT_MOD_INVALID ((1ULL<<56) - 1)
+#endif
 static_assert(NativePixmapPlane::kNoModifier == DRM_FORMAT_MOD_INVALID,
               "gfx::NativePixmapPlane::kNoModifier should be an alias for"
"DRM_FORMAT_MOD_INVALID");
```

**Why it does what it does :**
`DRM_FORMAT_MOD_INVALID` is part of the linux kernel and was introduced in 4.14 https://github.com/torvalds/linux/commit/e6fc3b68558e4c6d8d160b5daf2511b99afa8814#diff-94ab8a47ff02fa7f35cb0451290e1d85 this patch permit compatibility backward post 4.14

**Why** 
The patch is not really tied to opensuse... its why i suggested adding it to the portable

**Comment by Eloston** [comment]
I see. I'll include it then.

**Comment by Eloston** [comment]
Fixed with 94b7493c72a3a5767e8b1bca6c7f160f5449fa02
